dist/index.js

```diff
@@ -13,7 +13,6 @@ var $8EjUb$string_decoder = require("string_decoder");
 var $8EjUb$child_process = require("child_process");
 var $8EjUb$timers = require("timers");
 var $8EjUb$stream = require("stream");
-var $8EjUb$url = require("url");
```

Comment:
I'm not exactly sure what these other changes in the diff are.

I suspect it could be the result of some mismatch in Yarn versions.